### ISSUE TITLE
readme improvement on nodejs

### ISSUE THREAD

**Issue by tabetomo** [evanplaice/jquery-csv#68]
Hi, 

I"m a beginner on nodejs and jquery' and I spent some time for installing ... about require('./jquery.csv.js') for nodejs...

Then, it seems your jquery-csv exists as a npm package! Great.

but not mentioned.  So it could be easier if "npm import jquery-csv" and "require('jquery-csv')" is described somewhere in readme: e.g.

```
$ npm import jquery
$ npm import jquery-csv

var $ = jQuery = require('jquery');
require('jquery-csv'); 
```

Lastly your "The link to the stub: node-usage.js" is really helpful!! for me.

**Comment by innommable** [comment]
nice thank you i had the same issue this really helped.
don't know why none of this is mentioned in the original document, at least for beginners.

**Comment by evanplaice** [comment]
Try:

```javascript
var $ = jQuery = require('jquery');
$.csv = jQuery.csv = require('jquery-csv'); 
```

IIFE modules are pretty fragile in this regard. Transitioning to ESM is planned for a future update. I've been tracking progress on Node.js ESM support for a while now, looks like ESM sans .mjs is on its way to becoming a reality.

**Comment by evanplaice** [comment]
This is scheduled for the next minor release.

**Comment by evanplaice** [comment]
Fixed in 84ae464f62bab42069efb08f4d6ce18774220101
